scripts/userstyles.yml

```diff
@@ -292,6 +292,14 @@ userstyles:
       app-link: "https://github.com/libreddit/libreddit"
       current-maintainers: [*isabelroses]
       past-maintainers: [*andreasgrafen]
+  lichess.org:
```

Comment:
Hmm. Should we call it just "Lichess"? It seems to be consistently referred to as such on the [about page](https://lichess.org/about):

> In 2010, Thibault Duplessis began work on **Lichess** as a hobby project. The site was simple at the beginning, not even checking to see if moves were legal. He made the site [open source](https://www.youtube.com/watch?v=Tyd0FO0tko8), which means anyone is free to read the source code and make contributions. Gradually, the site improved and collected users as an enthusiastic volunteer staff assembled to help Thibault build and maintain the site.
> 
> Today, **Lichess** users play more than five million games every day. **Lichess** is one of the most popular chess websites in the world while remaining 100% free. Most “free” websites subsist by selling ads or selling user data. Others do it by putting all the good stuff behind paywalls. **Lichess** does none of these things and never will. With no investors demanding profits, **Lichess** staff can focus on improving the site as their only goal.

Reply:
Either way is good with me, i was going off the fact that their logo has .org, and how it's called .org in the more header parts of that same article

Reply:
Actually yeah, just lichess probably makes more sense